wp-content/themes/e-event/theme_config/demo_options.txt
a:2:{s:13:"theme_options";a:75:{s:10:"logo_image";s:75:"http://teslathemes.com/demo/wp/e-event/wp-content/uploads/2015/05/logow.png";s:9:"logo_text";s:0:"";s:15:"logo_text_color";s:0:"";s:14:"logo_text_font";s:0:"";s:14:"logo_text_size";s:0:"";s:17:"logo_wrapper_size";s:1:"3";s:11:"footer_logo";s:74:"http://teslathemes.com/demo/wp/e-event/wp-content/uploads/2015/05/logo.png";s:15:"home_logo_image";s:74:"http://teslathemes.com/demo/wp/e-event/wp-content/uploads/2015/05/logo.png";s:7:"favicon";s:77:"http://teslathemes.com/demo/wp/e-event/wp-content/uploads/2015/05/favicon.ico";s:10:"custom_css";s:0:"";s:9:"custom_js";s:0:"";s:8:"bg_color";s:0:"";s:8:"bg_image";s:0:"";s:10:"site_color";s:0:"";s:12:"site_color_2";s:0:"";s:15:"custom_selector";a:1:{i:0;s:0:"";}s:12:"custom_color";a:1:{i:0;s:0:"";}s:15:"custom_bg_color";a:1:{i:0;s:0:"";}s:10:"share_this";a:5:{i:0;s:10:"googleplus";i:1;s:8:"facebook";i:2;s:7:"twitter";i:3;s:9:"pinterest";i:4;s:8:"linkedin";}s:25:"social_platforms_facebook";s:36:"https://www.facebook.com/TeslaThemes";s:24:"social_platforms_twitter";s:31:"https://twitter.com/TeslaThemes";s:22:"social_platforms_vimeo";s:0:"";s:26:"social_platforms_pinterest";s:0:"";s:23:"social_platforms_google";s:0:"";s:24:"social_platforms_youtube";s:40:"http://www.youtube.com/user/TeslaThemes/";s:25:"social_platforms_linkedin";s:50:"http://www.linkedin.com/groups/TeslaThemes-5039596";s:25:"social_platforms_dribbble";s:31:"http://dribbble.com/TeslaThemes";s:20:"social_platforms_rss";s:0:"";s:26:"social_platforms_instagram";s:0:"";s:13:"tracking_code";s:0:"";s:19:"twitter_consumerkey";s:22:"P9x4zzKjzteRTk5YX1J0KA";s:22:"twitter_consumersecret";s:43:"WOyfbMUNvTJRc5AhrHspyLZoPVbHtQnVokvECXSWkoU";s:19:"twitter_accesstoken";s:50:"1427423114-YVeKoZIBrZnL9U70Umx1EAXwNajPB0ow7UaRmud";s:25:"twitter_accesstokensecret";s:45:"l28FF9fYatMYA1GI3KAfAy2mAZrGXbLGwGVCmjbFe5nui";s:11:"error_title";s:3:"404";s:13:"error_message";s:20:"ERROR PAGE NOT FOUND";s:16:"error_back_color";s:7:"#262626";s:12:"error_button";s:7:"Go Home";s:11:"error_image";s:78:"http://teslathemes.com/demo/wp/e-event/wp-content/uploads/2015/05/error-bg.jpg";s:14:"show_countdown";s:14:"show_countdown";s:20:"show_countdown_title";s:28:"<span>Event</span> starts in";s:12:"footer_style";s:1:"1";s:16:"show_map_contact";s:16:"show_map_contact";s:17:"copyright_message";s:93:"Copyright 2015 Design by <a href="http://www.teslathemes.com" target="_blank">TeslaThemes</a>";s:17:"header_background";s:77:"http://teslathemes.com/demo/wp/e-event/wp-content/uploads/2015/05/path-bg.jpg";s:19:"show_related_events";s:19:"show_related_events";s:18:"show_related_title";s:14:"Related Events";s:19:"show_related_number";s:1:"3";s:18:"contact_map_coords";s:36:"48.84261087702928, 2.408274378776582";s:25:"contact_map_marker_coords";s:35:"48.84825933106276,2.392073836326631";s:16:"contact_map_zoom";s:2:"13";s:16:"contact_map_icon";s:145:"http://teslathemes.com/demo/wp/e-event/wp-content/themes/e-event/tesla_framework/static/images/mapicons/../../../../theme_config/icons/marker.png";s:22:"contact_map_mapOptions";s:51:"styles : [{stylers: [     { saturation: -80 }   ]}]";s:13:"email_contact";s:19:"contact@e-event.com";s:13:"contact_phone";s:12:"+341 234 412";s:25:"contact_information_title";s:19:"Contact Information";s:24:"contact_information_text";s:367:"These cases are perfectly simple and easy to distinguish and when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every pain avoided. The wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other greater pleasures, or else he endures pains to avoid worse pains.";s:21:"contact_offices_title";s:15:"Office Adresses";s:15:"contact_offices";s:42:"Jacksonville, Illinois
443 - Main Street.";s:27:"contact_phone_numbers_title";s:13:"Phone Numbers";s:21:"contact_phone_numbers";s:30:"+09 202 02 020
+05 303 03 030";s:26:"contact_email_adress_title";s:14:"Contact Emails";s:20:"contact_email_adress";s:39:"contact@e-event.com
events@e-event.com";s:19:"contact_header_text";s:26:"Don't miss the next event!";s:20:"contact_header_phone";s:12:"+341 234 412";s:20:"contact_header_email";s:19:"contact@e-event.com";s:23:"main_content_text_color";s:0:"";s:22:"main_content_text_font";s:0:"";s:22:"main_content_text_size";s:0:"";s:18:"sidebar_text_color";s:0:"";s:17:"sidebar_text_font";s:0:"";s:17:"sidebar_text_size";s:0:"";s:15:"menu_text_color";s:0:"";s:14:"menu_text_font";s:0:"";s:14:"menu_text_size";s:0:"";}s:12:"form_builder";a:3:{i:0;a:5:{s:2:"id";s:1:"1";s:14:"receiver_email";s:15:"tesla@gmail.com";s:15:"disable_headers";s:0:"";s:8:"location";s:12:"contact_page";s:4:"rows";a:2:{i:0;a:1:{s:7:"columns";a:2:{i:0;a:2:{s:13:"form_elements";a:1:{i:0;a:6:{s:4:"name";s:12:"contact-name";s:4:"type";s:4:"text";s:5:"title";s:4:"Text";s:11:"placeholder";s:4:"Name";s:5:"label";s:0:"";s:8:"required";s:4:"true";}}s:4:"size";s:1:"6";}i:1;a:2:{s:13:"form_elements";a:1:{i:0;a:6:{s:4:"name";s:13:"contact-email";s:4:"type";s:5:"email";s:5:"title";s:5:"Email";s:11:"placeholder";s:6:"E-mail";s:5:"label";s:0:"";s:8:"required";s:4:"true";}}s:4:"size";s:1:"6";}}}i:1;a:1:{s:7:"columns";a:1:{i:0;a:2:{s:13:"form_elements";a:3:{i:0;a:6:{s:4:"name";s:15:"contact-website";s:4:"type";s:4:"text";s:5:"title";s:4:"Text";s:11:"placeholder";s:7:"Website";s:5:"label";s:0:"";s:8:"required";s:5:"false";}i:1;a:6:{s:4:"name";s:15:"contact-message";s:4:"type";s:8:"textarea";s:5:"title";s:8:"Textarea";s:11:"placeholder";s:7:"Message";s:5:"label";s:0:"";s:8:"required";s:4:"true";}i:2;a:3:{s:4:"type";s:6:"submit";s:5:"title";s:6:"Submit";s:5:"label";s:6:"Submit";}}s:4:"size";s:2:"12";}}}}}i:1;a:5:{s:2:"id";s:1:"2";s:14:"receiver_email";s:15:"tesla@gmail.com";s:15:"disable_headers";s:0:"";s:8:"location";s:0:"";s:4:"rows";a:1:{i:0;a:1:{s:7:"columns";a:1:{i:0;a:2:{s:13:"form_elements";a:5:{i:0;a:6:{s:4:"name";s:4:"name";s:4:"type";s:4:"text";s:5:"title";s:4:"Text";s:11:"placeholder";s:9:"Your Name";s:5:"label";s:0:"";s:8:"required";s:4:"true";}i:1;a:6:{s:4:"name";s:5:"email";s:4:"type";s:4:"text";s:5:"title";s:4:"Text";s:11:"placeholder";s:14:"E-mail Address";s:5:"label";s:0:"";s:8:"required";s:4:"true";}i:2;a:6:{s:4:"name";s:5:"phone";s:4:"type";s:4:"text";s:5:"title";s:4:"Text";s:11:"placeholder";s:12:"Phone Number";s:5:"label";s:0:"";s:8:"required";s:4:"true";}i:3;a:2:{s:4:"type";s:13:"current_event";s:5:"title";s:27:"Events (only on event page)";}i:4;a:3:{s:4:"type";s:8:"submit_2";s:5:"title";s:8:"Submit 2";s:5:"label";s:8:"register";}}s:4:"size";s:2:"12";}}}}}i:2;a:5:{s:2:"id";s:1:"4";s:14:"receiver_email";s:0:"";s:15:"disable_headers";s:0:"";s:8:"location";s:12:"contact_page";s:4:"rows";a:1:{i:0;a:1:{s:7:"columns";a:2:{i:0;a:2:{s:13:"form_elements";a:7:{i:0;a:3:{s:4:"type";s:13:"new_form_rsvp";s:5:"title";s:48:"Use other form style? drag this anywhere in form";s:5:"label";s:4:"RSVP";}i:1;a:6:{s:4:"name";s:10:"first-name";s:4:"type";s:4:"text";s:5:"title";s:4:"Text";s:11:"placeholder";s:10:"First Name";s:5:"label";s:0:"";s:8:"required";s:5:"false";}i:2;a:6:{s:4:"name";s:9:"last-name";s:4:"type";s:4:"text";s:5:"title";s:4:"Text";s:11:"placeholder";s:9:"Last Name";s:5:"label";s:0:"";s:8:"required";s:5:"false";}i:3;a:6:{s:4:"name";s:5:"email";s:4:"type";s:5:"email";s:5:"title";s:5:"Email";s:11:"placeholder";s:14:"E-mail address";s:5:"label";s:0:"";s:8:"required";s:4:"true";}i:4;a:6:{s:4:"name";s:5:"phone";s:4:"type";s:4:"text";s:5:"title";s:4:"Text";s:11:"placeholder";s:5:"Phone";s:5:"label";s:0:"";s:8:"required";s:5:"false";}i:5;a:4:{s:4:"name";s:7:"respond";s:4:"type";s:5:"radio";s:5:"title";s:5:"Radio";s:5:"label";s:20:"YES, I AM ATTENDING.";}i:6;a:4:{s:4:"name";s:7:"respond";s:4:"type";s:5:"radio";s:5:"title";s:5:"Radio";s:5:"label";s:22:"NO,I AM NOT ATTENDING.";}}s:4:"size";s:1:"6";}i:1;a:2:{s:13:"form_elements";a:3:{i:0;a:2:{s:4:"type";s:13:"current_event";s:5:"title";s:27:"Events (list of all events)";}i:1;a:6:{s:4:"name";s:7:"message";s:4:"type";s:8:"textarea";s:5:"title";s:8:"Textarea";s:11:"placeholder";s:7:"Message";s:5:"label";s:0:"";s:8:"required";s:4:"true";}i:2;a:3:{s:4:"type";s:6:"submit";s:5:"title";s:6:"Submit";s:5:"label";s:5:"Write";}}s:4:"size";s:1:"6";}}}}}}}

wp-content/themes/e-event/js/options.js
(function () {

    "use strict";

    var matched, browser;

    // Use of jQuery.browser is frowned upon.
    // More details: http://api.jquery.com/jQuery.browser
    // jQuery.uaMatch maintained for back-compat
    jQuery.uaMatch = function (ua) {
        ua = ua.toLowerCase();

        var match = /(chrome)[ \/]([\w.]+)/.exec(ua) ||
            /(webkit)[ \/]([\w.]+)/.exec(ua) ||
            /(opera)(?:.*version|)[ \/]([\w.]+)/.exec(ua) ||
            /(msie) ([\w.]+)/.exec(ua) ||
            ua.indexOf("compatible") < 0 && /(mozilla)(?:.*? rv:([\w.]+)|)/.exec(ua) || [];

        return {
            browser: match[1] || "",
            version: match[2] || "0"
        };
    };

    matched = jQuery.uaMatch(navigator.userAgent);
    browser = {};

    if (matched.browser) {
        browser[matched.browser] = true;
        browser.version = matched.version;
    }

    // Chrome is Webkit, but Webkit is also Safari.
    if (browser.chrome) {
        browser.webkit = true;
    } else if (browser.webkit) {
        browser.safari = true;
    }

    jQuery.browser = browser;

    jQuery.sub = function () {
        function jQuerySub(selector, context) {
            return new jQuerySub.fn.init(selector, context);
        }
        jQuery.extend(true, jQuerySub, this);
        jQuerySub.superclass = this;
        jQuerySub.fn = jQuerySub.prototype = this();
        jQuerySub.fn.constructor = jQuerySub;
        jQuerySub.sub = this.sub;
        jQuerySub.fn.init = function init(selector, context) {
            if (context && context instanceof jQuery && !(context instanceof jQuerySub)) {
                context = jQuerySub(context);
            }

            return jQuery.fn.init.call(this, selector, context, rootjQuerySub);
        };
        jQuerySub.fn.init.prototype = jQuerySub.fn;
        var rootjQuerySub = jQuerySub(document);
        return jQuerySub;
    };

})();

/* =================Twitter============================ */
var load_twitter = function () {
    "use strict";
    var linkify = function (text) {
        text = text.replace(/(https?:\/\/\S+)/gi, function (s) {
            return '<a href="' + s + '">' + s + '</a>';
        });
        text = text.replace(/(^|)@(\w+)/gi, function (s) {
            return '<a href="http://twitter.com/' + s + '">' + s + '</a>';
        });
        text = text.replace(/(^|)#(\w+)/gi, function (s) {
            return '<a href="http://search.twitter.com/search?q=' + s.replace(/#/, '%23') + '">' + s + '</a>';
        });
        return text;
    };
    jQuery('.twitter_widget').each(function () {
        var t = jQuery(this);
        var t_date_obj = new Date();
        var t_loading = 'Loading tweets..'; //message to display before loading tweets
        var t_container = jQuery('<ul>').addClass('twitter').append('<li>' + t_loading + '</li>');
        t.append(t_container);
        var t_user = t.attr('data-user');
        var t_posts = parseInt(t.attr('data-posts'), 10);
        jQuery.getJSON("php/twitter.php?user=" + t_user, function (t_tweets) {
            t_container.empty();
            for (var i = 0; i < t_posts && i < t_tweets.length; i++) {
                var t_date = Math.floor((t_date_obj.getTime() - Date.parse(t_tweets[i].created_at)) / 1000);
                var t_date_str;
                var t_date_seconds = t_date % 60;
                t_date = Math.floor(t_date / 60);
                var t_date_minutes = t_date % 60;
                if (t_date_minutes) {
                    t_date = Math.floor(t_date / 60);
                    var t_date_hours = t_date % 60;
                    if (t_date_hours) {
                        t_date = Math.floor(t_date / 60);
                        var t_date_days = t_date % 24;
                        if (t_date_days) {
                            t_date = Math.floor(t_date / 24);
                            var t_date_weeks = t_date % 7;
                            if (t_date_weeks)
                                t_date_str = t_date_weeks + ' week' + (1 == t_date_weeks ? '' : 's') + ' ago';
                            else
                                t_date_str = t_date_days + ' day' + (1 == t_date_days ? '' : 's') + ' ago';
                        } else
                            t_date_str = t_date_hours + ' hour' + (1 == t_date_hours ? '' : 's') + ' ago';
                    } else
                        t_date_str = t_date_minutes + ' minute' + (1 == t_date_minutes ? '' : 's') + ' ago';
                } else
                    t_date_str = t_date_seconds + ' second' + (1 == t_date_seconds ? '' : 's') + ' ago';
                var t_message =
                    '<li>' +
                    linkify(t_tweets[i].text) +
                    '<span>' +
                    t_date_str +
                    '</span>' +
                    '</li>';
                t_container.append(t_message);
            }
        });
    });
};
//load modules-------------

/*  SCROOL TOP  */
jQuery(document).ready(function () {
    "use strict";

    jQuery('.scroll-top').click(function () {
        jQuery('body,html').animate({
            scrollTop: 0
        }, 1200, 'swing');
        return false;
    });

    jQuery(function() {
        if(jQuery('.sticky-bar').length) {
            jQuery(".sticky-bar").sticky({topSpacing:0});       
        }
    });

    jQuery(".responsive-menu").click(function (e) {
        jQuery(".main-nav>ul").toggle();
        e.stopPropagation();
        if (e.preventDefault)
            e.preventDefault();
        return false;
    });
    jQuery("body").click(function () {
        jQuery(".main-nav>ul").hide();
    });

    if (!Array.prototype.indexOf) {
        Array.prototype.indexOf = function (obj, start) {
            for (var i = (start || 0), j = this.length; i < j; i++) {
                if (this[i] === obj) {
                    return i;
                }
            }
            return -1;
        };
    }
    /* GALLERY IMAGE ZOOM */
    jQuery(".swipebox").swipebox();

    jQuery('input[placeholder], textarea[placeholder]').placeholder();

    load_twitter();

    jQuery('header .right-box .fa-search').on('click', function () {
            jQuery('.search-container').addClass('open');
    });
    jQuery('.search-container .close-icon').on('click', function () {
            jQuery('.search-container').removeClass('open');
    });

    jQuery('.search-container .search-box .search-form .search-input').keyup(function () {
        var text_val = jQuery(this).val();
        if (text_val === "") {
            jQuery('.search-container .search-box .search-form').removeClass('has-value');
        } else {
            jQuery('.search-container .search-box .search-form').addClass('has-value');
        }
    });

    jQuery('.the-countdown').each(function(){
        var cd_duedate = jQuery(this).attr('data-duedate');
        var cd_start = new Date().getTime();
        var cd_end = new Date(cd_duedate).getTime();

        jQuery(this).countdown(cd_duedate, function(event) {
            var $this = jQuery(this);
            // Total days
            var days = Math.round(Math.abs((cd_start - cd_end))/(24*60*60*1000));
            var divider = {
                'seconds':60,
                'minutes':60,
                'hours':24
            };
            var progress = null;
            switch (event.type) {
                case "seconds":
                case "minutes":
                case "hours":
                case "days":
                case "weeks":
                case "daysLeft":
                    $this.find('#' + event.type).html(event.value);
                    if(event.type === 'days'){
                        progress = ((days - event.value) * 100) / (days);
                    }else{                    
                        progress = (100 / divider[event.type]) * (divider[event.type] - event.value);
                    }
                    break;
                case "finished":
                    $this.hide();
                    break;
            }
        });
    });

    jQuery('.check-value').on('focusout', function () {
        var text_val = jQuery(this).val();
        if (text_val === "" || text_val.replace(/^\s+|\s+$/g, '') === "") {
            jQuery(this).removeClass('has-value');
        } else {
            jQuery(this).addClass('has-value');
        }
    });


    jQuery('.check-value').each(function(){
        var text_val = jQuery(this).val();

        if (text_val === "" || text_val.replace(/^\s+|\s+$/g, '') === "") {
            jQuery(this).removeClass('has-value');
        } else {
            jQuery(this).addClass('has-value');
        }
    });

});